src/bp/ui-studio/src/web/reducers/flows.ts

```diff
@@ -547,25 +539,36 @@ reducer = reduceReducers(
           node: ''
         }))
 
-        const updatedNodes = state.flowsByName[parentFlowName].nodes.map(node => {
-          if (node.flow !== activeFlow) {
-            return node
+        const updatedFlows = {}
+        for (const [name, flow] of Object.entries(state.flowsByName)) {
+          if (!flow.nodes.find(n => n.flow === activeFlow)) {
+            continue
           }
```

Comment:
You could replace this by the selector I suggested you earlier

Reply:
I don't think I have access to the RootReducer from here